### PR TITLE
Add nf-core governance page

### PR DESCRIPTION
**Pull request by christopher-hakkaart**
Add nf-core governance page

<a href="https://gitpod.io/#https://github.com/nf-core/nf-co.re/pull/1440"></a>

### CODE REVIEW

File: public_html/governance.php [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?php
+$title = 'Governance';
+$subtitle = 'How nf-core is governed';
+$markdown_fn = '../markdown/governance.md';
+$md_github_url = 'https://github.com/nf-core/nf-co.re/blob/master/markdown/governance.md';
+include '../includes/header.php';
```
Comment (mashehu) [inline public_html/governance.php:6]:
```suggestion
$include_mermaid = true;
include '../includes/header.php';
```

File: includes/header.php [hunk 2]
```diff
@@ -57,6 +57,7 @@
   <script src="/assets/lib/bootstrap.bundle.min.js"></script>
   <script src="/assets/lib/highlight.min.js"></script>
   <script src="/assets/lib/groovy.min.js"></script>
+  <script src="/assets/lib/mermaid.js"></script>
```
Comment (mashehu) [inline includes/header.php:60]: move this to page-specific js and check for $includes_mermaid I added to governance.php

File: markdown/governance.md [hunk 3]
```diff
@@ -0,0 +1,220 @@
...
+
+The outreach team will have one or more leads who are responsible for overseeing outreach efforts.
+New members will be invited to be a part of the outreach team based on experience and outreach activity (e.g., involvement in the ambassador program). 
+The outreach team is not a fixed size and will scale as the community grows.
+The outreach leads will have access to community social media and YouTube accounts (e.g., Twitter and YouTube).
+
+**Responsibilities**
+
+- Organizing and running the `#bytesize` seminar series
+- Leading the organizing hackathon, training, and other outreach events
```
Comment (mribeirodantas) [inline markdown/governance.md:143]:
```suggestion
- Leading the organization of hackathons, training sessions, and other outreach events
```

File: markdown/governance.md [hunk 4]
```diff
@@ -0,0 +1,220 @@
...
+Community members with an interest in sharing knowledge and helping their communities can become nf-core ambassadors.
+Ambassadors are expected to regularly engage with the wider community and will be supported by the outreach team and other ambassadors.
+
+More information about becoming an nf-core ambassador can be found [here](https://github.com/).
+
+**Responsibilities**
+
+- Engaging with local, national, and international Nextflow and nf-core community
+- Writing or sharing tweets, blogs, articles, and technical documentation
+- Running nf-core events such as user groups, hackathons, workshops, and more
```
Comment (mribeirodantas) [inline markdown/governance.md:208]:
```suggestion
- Running nf-core events such as user group meetings, hackathons, workshops, and more
```

**Review by mribeirodantas — COMMENTED** [review]
I think it's pretty good so far, @christopher-hakkaart !

There is probably more to add in the ambassador section but for now, I think it's enough!

**Comment by ewels** [comment]
@nf-core-bot fix linting

**Comment by maxulysse** [comment]
I'm not sure this one is up to date anymore

File: markdown/governance.md [hunk 5]
```diff
@@ -0,0 +1,244 @@
...
+The outreach leads will have access to community social media and YouTube accounts (e.g., Twitter and YouTube).
+
+**Responsibilities**
+
+- Organizing and running the `#bytesize` seminar series
+- Leading the organization of hackathons, training sessions, and other outreach events
+- Organizing and running the ambassador program
+- Creating and sharing community content
+- Regular attendance at outreach team meetings
+
```
Comment (FranBonath) [inline markdown/governance.md:147]: what about the mentorship program? Should this be mentioned here?

**Review by FranBonath — COMMENTED** [review]
There is still info about the ambassadors missing. Could we remove or amend that to have a mergable version before we move to the new website?

**Comment by ewels** [comment]
> There is still info about the ambassadors missing.

We don't have an ambassadors program yet, that's why 😄 It's being actively worked on currently, hoping to have something to announce for the summit, will add detail (and ask for feedback) when the plan is a bit more solid.

Agree that we should add a mention of this fact here though, as we talk about the program a fair bit. Instead of removing those mentions, adding a note that it's coming soon should suffice I think.

> what about the mentorship program? Should this be mentioned here?

Mentorship program is a bit up in the air currently, as we don't have funding for any more rounds. It's an open question as to whether we want to continue trying to run it without any payment for mentors. I guess it doesn't hurt to add it 🤷🏻

**Pull request merged**